scripts/merge_lora.py

```diff
@@ -18,66 +20,75 @@
 
 
 def merge_lora(
-    lora_path: Path = Path("out/lora/alpaca/lit_model_lora_finetuned.pth"),
-    checkpoint_dir: Path = Path("checkpoints/stabilityai/stablelm-base-alpha-3b"),
-    out_dir: Path = Path("out/lora/checkpoint"),
+    checkpoint_dir: Path,
+    pretrained_checkpoint_dir: Optional[Path] = None,
     precision: Optional[str] = None,
-    lora_r: int = 8,
-    lora_alpha: int = 16,
-    lora_dropout: float = 0.05,
-    lora_query: bool = True,
-    lora_key: bool = False,
-    lora_value: bool = True,
-    lora_projection: bool = False,
-    lora_mlp: bool = False,
-    lora_head: bool = False,
 ) -> None:
-    """Generates a response based on a given instruction and an optional input.
-    This script will only work with checkpoints from the instruction-tuned GPT-LoRA model.
-    See `finetune/lora.py`.
+    """Merges the LoRA weights with the base model. See `finetune/lora.py`.
+
+    Merging happens in-place in the checkpoint directory that is given as input.
```

Comment:
Can you mention here that a copy of the lora weights is generated (and where to)